app/controllers/configuration_controller.rb

```diff
@@ -11,6 +12,10 @@ class ConfigurationController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  def title
+    session[:config_title] ? _("configuration") : session[:config_title]
```

Comment:
Also.. looking for `config_title` in the code, this is the only place it's ever used...

```
$ ag config_title
app/controllers/configuration_controller.rb
611:    @title        = session[:config_title] ? _("Configuration") : session[:config_title]
```

So you can safely assume that it won't ever be set and just return `_("Configuration")` always.